app/views/search/index.html.slim

```diff
@@ -9,10 +9,14 @@
           .visible-xs
             = render template: "search/_mobile_categories.html.slim"
       .col-xs-12.col-md-8.col-lg-9.search.search-result
-        - case params[:type]
-          - when 'repositories'
-          = render template: "search/_repositories.html.slim"
-          -when 'namespaces'
-          = render template: "search/_namespaces.html.slim"
-          -when 'teams'
-          = render template: "search/_teams.html.slim"
+        .search-filter
```

Comment:
This form has one hidden field (type) that won't be repeated on the others places with the dynamic field...
Maybe the helper would be better for these places. What you think?